ReportPortal.ATM/TestProject/Pages/AddNewWidgetPage/AddNewWidgetPage.private.cs

```diff
@@ -0,0 +1,77 @@
+﻿using OpenQA.Selenium;
+using System;
+using WebDriverLibrary.Extensions.WebDrivers;
+
+namespace TestProject.Pages.AddNewWidgetPage;
+
+public partial class AddNewWidgetPage
+{
+    private void SelectWidget(string type)
+    {
+        try
+        {
+            WidgetTypeInput(type).Click();
+        }
+        catch (Exception e) { throw; }
+    }
```

Comment:
Once you apply the suggestion on the private method, you can just receive a string type, and pass it to the private method to select the desired widget.